Adding HCP and HCP Terraform Links to make the platform easier to find (#856)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/deletion.mdx

After HCP Terraform completes a manual or automatic destroy run, it waits until further state updates to schedule a new auto-destroy run.

> To remove your workspace's auto-destroy based on inactivity:

Reviewer: ```suggestion
To remove your workspace's auto-destroy run:
```

> 1. Sign in to [HCP Terraform](https://app.terraform.io/) or Terraform Enterprise and navigate to the workspace you no longer want to auto-destroy.

Reviewer: ```suggestion
1. Sign in to [HCP Terraform](https://app.terraform.io/) or Terraform Enterprise and navigate to the workspace you want to disable the auto-destroy run for.
```
